protocol/v2/blockchain/beacon/network.go

```diff
@@ -11,17 +11,38 @@ import (
 // Network is a beacon chain network.
 type Network struct {
 	spectypes.BeaconNetwork
+	LocalTestNet bool
 }
 
 // NewNetwork creates a new beacon chain network.
 func NewNetwork(network spectypes.BeaconNetwork) Network {
-	return Network{network}
+	return Network{
+		BeaconNetwork: network,
+		LocalTestNet:  false,
+	}
+}
+
+// NewNetworkWithLocalTestNet creates a new beacon chain network.
+func NewNetworkWithLocalTestNet(network spectypes.BeaconNetwork, localTestNet bool) Network {
```

Comment:
I think we can use an optional pattern and pass this option to NewNetwork.

Example: https://golang.cafe/blog/golang-functional-options-pattern.html

Reply:
Or just 
```
func NewNetworkWithLocalTestNet(network spectypes.BeaconNetwork) Network
``` 
and set `LocalTestNet` to `true` because `NewNetwork` sets it to false